_posts/0000-01-17-bettumgal.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+DevOps Journey Starts here!! Third edit
```

Comment:
Exciting!

Now we need to figure out calculi and Guide Rails 😅 

May you can add an emoji to capture the excitement